api/src/main/java/org/apache/iceberg/Metrics.java

```diff
@@ -37,6 +38,7 @@ public class Metrics implements Serializable {
   private Map<Integer, Long> nanValueCounts = null;
   private Map<Integer, ByteBuffer> lowerBounds = null;
   private Map<Integer, ByteBuffer> upperBounds = null;
+  private Map<Integer, Type> originalTypes = null;
```

Comment:
Do we want to comment somewhere that the `originalTypes` are not serialized?

Reply:
yep good idea